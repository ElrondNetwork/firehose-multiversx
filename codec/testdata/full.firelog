FIRE BLOCK_BEGIN 6
FIRE BLOCK_END 6 ef2d127de37b942baad06145e54b0c619a1f22327b2ebbcfbec78f5564afe39d 1647087903319636000 12096d657461636861696e1a04686173682a0d0a0665726431717110041a0100