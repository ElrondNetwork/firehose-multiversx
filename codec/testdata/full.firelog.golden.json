[{
  "height": 6,
  "prevHash": "ef2d127de37b942baad06145e54b0c619a1f22327b2ebbcfbec78f5564afe39d",
  "timestamp": 1647087903319636000,
  "MultiversxBlock": {
    "HeaderType": "metachain",
    "HeaderHash": "aGFzaA==",
    "AlteredAccounts": [
      {
        "Address": "erd1qq",
        "Nonce": 4,
        "Balance": "100"
      }
    ]
  }
}
]